components/page-header/page-header.component.ts

```diff
@@ -131,7 +131,7 @@ export class NzPageHeaderComponent implements AfterViewInit, OnDestroy, OnInit {
   }
 
   ngAfterViewInit(): void {
-    if (!this.nzBack.observed) {
+    if (this.nzBack.observers.length) {
```

Comment:
!observed <=equals=> !observers.length

```suggestion
    if (!this.nzBack.observers.length) {
```

Reply:
True my bad

Reply:
done